config/util.go

```diff
@@ -81,3 +82,30 @@ func SetIfNotDefault(src interface{}, dest interface{}) {
 		}
 	}
 }
+
+// DurationOpt provides a datatype to use with ParseDurations
+type DurationOpt struct {
+	// The duration we need to parse
+	Duration string
+	// Where to store the result
+	Dst *time.Duration
+	// A variable name associated to it for helpful errors.
+	Name string
+}
+
+// ParseDurations takes a time.Duration src and saves it to the given dst. into the given
```

Comment:
Remove leftover comment on the end of the line, `into the given`.

Reply:
ok